string_search/string_search.java

```diff
@@ -0,0 +1,36 @@
+package string_search;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class string_search {
+
+    public static void main(String[] args) {
+        // System.out.println(args[0]);
+        Scanner scan = new Scanner(System.in);
+        String fileName;
+        System.out.println("Enter file name: ");
+        fileName = scan.next();
+        System.out.println("Enter searchword: ");
+        String search = scan.next();
+        if (!fileName.contains("txt")) {
+            fileName += ".txt";
```

Comment:
Don't really understand the purpose of this code. Also, should other files than txt not be allowed?

Reply:
Now removed